lang/java/reef-webserver/src/main/java/org/apache/reef/webserver/HttpServerImpl.java

```diff
@@ -109,15 +116,15 @@
   private Server tryPort(final int portNumber) throws Exception {
     Server srv = new Server();
     final Connector connector = new SocketConnector();
-    connector.setHost(addressProvider.getLocalAddress());
+    connector.setHost(this.hostAddress);
     connector.setPort(portNumber);
     srv.addConnector(connector);
     try {
       srv.start();
       LOG.log(Level.INFO, "Jetty Server started with port: {0}", portNumber);
     } catch (final BindException ex) {
       srv = null;
-      LOG.log(Level.FINEST, "Cannot use port: {0}. Will try another", portNumber);
+      LOG.log(Level.INFO, String.format("Cannot use host:%s,port: %s. Will try another", this.hostAddress, portNumber));
```

Comment:
>INFO [](start = 20, length = 4)

Change back to FINEST before sending pull request to apache/master. I left it here as it is helps me in debugging the containers feature.